testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -62,7 +62,7 @@ class MiniAppDisplayActivity : BaseActivity() {
 
             launch {
                 viewModel.obtainMiniAppView(appId, object: MiniAppMessageBridge() {
-                    override fun getUniqueId() = "example_unique_id"
+                    override fun getUniqueId() = UUID.nameUUIDFromBytes(Build.DEVICE.toByteArray()).toString()
```

Comment:
people with the same phone will have the UUID generated from the same array ?

Reply:
That's right. There is no authentication for different users in our sample app at the moment so the sample app uses uuid generated from device.

Reply:
Maybe we could use the Android ID instead? https://developer.android.com/reference/android/provider/Settings.Secure#ANDROID_ID